src/functions-and-arrays.js

```diff
@@ -52,16 +101,34 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
+function uniquifyArray(arr) {
+  if (arr.length === 0) {
+    return null;
+  }
```

Comment:
For shorter syntax, if you return only one line of code, you can write it like this

```
if (arr.length === 0) return null;
```